web_src/js/markup/mermaid.js

```diff
@@ -4,9 +4,11 @@ import {displayError} from './common.js';
 
 const {mermaidMaxSourceCharacters} = window.config;
 
+// margin removal is for https://github.com/mermaid-js/mermaid/issues/4907
 const iframeCss = `:root {color-scheme: normal}
 body {margin: 0; padding: 0; overflow: hidden}
-#mermaid {display: block; margin: 0 auto}`;
+#mermaid {display: block; margin: 0 auto}
+blockquote, dd, dl, figure, h1, h2, h3, h4, h5, h6, hr, p, pre {margin: 0}`;
```

Comment:
Shouldn't this be

`#mermaid blockquote ,#mermaid dd, #mermaid dl, #mermaid figure, #mermaid h1, #mermaid h2, #mermaid h3, #mermaid h4, #mermaid h5, #mermaid h6, #mermaid hr, #mermaid p, #mermaid pre { margin: 0}`

Reply:
This CSS is already contained in an iframe, so there is no risk of the rules affecting the outside page. Adding these would likely introduce more issues because of CSS specificity. The [mermaid demo](https://mermaid.live) has the exact same rules, too.

Reply:
ah sorry I missed that!